Makefile

```diff
@@ -6,10 +6,15 @@ all: libvtile
 ./deps/gyp:
 	git clone https://chromium.googlesource.com/external/gyp.git ./deps/gyp && cd ./deps/gyp && git checkout 3464008
 
+./deps/pbf:
+	git clone https://github.com/mapbox/pbf.hpp.git ./deps/pbf && cd ./deps/pbf && git checkout 0d1c0061e4
+
+build/Makefile: ./deps/pbf ./deps/gyp gyp/build.gyp test/*cpp
```

Comment:
^^ looks like that snuck in from the merge. If I'm reading the diff right there is another (correct) `build/Makefile` line down below.